lazy-imports.html

```diff
@@ -0,0 +1,64 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+`lazy-imports`
+Declarative lazy imports
+
+@demo demo/index.html
+-->
+<script>
+
+/** @polymerBehavior */
+var LazyImportsBehavior = {
+  __loadedImports: {},
+  importLazyGroup: function(group) {
+    var domModule = Polymer.DomModule.import(this.localName);
+    var groupAttribute = '';
+    if (group) {
+      groupAttribute = `[group=${group}]`;
+    }
+    var query = `link${groupAttribute}[rel='lazy-import'][href]`
```

Comment:
This could be 
```javascript
var query = `link${groupAttribute}[rel='lazy-import'][href]:not([href=""])`
```
That should allow you to remove lines 35 to 37 🎉